src/Custom/test/CustomCalculatorGTest.cpp

```diff
@@ -0,0 +1,108 @@
+//////////////////////////////////////////////////////////////////////////////
+/**
+ *  @file   CustomCalculatorGTest.cpp
+ *  @date   Thu, June 10 2021
+ *  @brief  Tests for CustomCalculator
+ */
+////////////////////////////////////////////////////////////////////////////
+
+
+#include<gtest/gtest.h>
+#include<gmock/gmock.h>
+
+#include "CustomCalculator.h"
+
+using namespace testing;
+using namespace calculator;
+
+static bool isEqual(double a, double b)
+{
+    return fabs(a - b) < DBL_EPSILON;
+}
+
+/**
+ * @brief simple fixture for CustomCalculator for simple functions
+ */
+class CustomCalculatorTests: public ::testing::Test
+{
+    protected:
+        std::function<int(std::vector<int>)> sumInts =
+                [](const std::vector<int>& terms)
+                {
+                    int sum = 0;
+                    for(int term : terms)
+                    {
+                        sum += term;
+                    }
+                    return sum;
+                };
+        std::function<double(std::vector<double>)> sumDoubles =
+                [](const std::vector<double>& terms)
+                {
+                    double sum = 0;
+                    for(double term : terms)
+                    {
+                        sum += term;
+                    }
+                    return sum;
+                };
+};
+
+/**
+ * @brief Positive case to make sure result is correct
+ */
+TEST_F(CustomCalculatorTests, WhenGetResultIsCalledWithCustomCalculator_ThenAnswerReturned)
+{
+    CustomCalculator<int> calculator1{{1, 2, 3}, sumInts, "{} + {} + {}"};
```

Comment:
can create a local string for this unit test to use this string so you can reuse it later "{} + {} + {}"